package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "etl-toolkit",
+  "version": "0.1.0",
+  "description": "general components for managing the retrieval and processing of data",
+  "main": "etlToolkit.js",
+  "repository": "https://github.com/nhsuk/etl-toolkit",
+  "author": "mark <markharrop@yahoo.com>",
+  "license": "MIT",
+  "private": false,
+  "peer-dependencies": {
+    "async": "^2.6.0",
+    "nhsuk-bunyan-logger": "^1.8.0"
+  },
+  "dependencies": {
+    "async": "^2.6.0",
+    "nhsuk-bunyan-logger": "^1.8.0"
+  },
+  "scripts": {
+    "coverage-check": "istanbul check-coverage --config .istanbul.yml",
+    "coverage-generate": "yarn istanbul cover -- _mocha -- --recursive test",
+    "coverage-upload-coveralls": "cat ./coverage/lcov.info | coveralls",
+    "git-hook": "yarn lint && yarn coverage-generate && yarn coverage-check",
+    "istanbul": "NODE_ENV=test istanbul",
+    "lint": "eslint --ext .js,.json .",
+    "postrewrite": "yarn git-hook",
+    "precommit": "yarn git-hook",
+    "prepush": "yarn git-hook",
+    "test": "NODE_ENV=test mocha --recursive test",
+    "test-ci": "yarn lint && yarn coverage-generate && yarn coverage-check && yarn coverage-upload-coveralls",
+    "test-integration": "NODE_ENV=test mocha --recursive test/integration",
+    "test-unit": "NODE_ENV=test mocha --recursive test/unit",
+    "test-watch": "yarn test --watch --reporter min",
+    "test-watch-integration": "yarn test-integration --watch --reporter min",
+    "test-watch-unit": "yarn test-unit --watch --reporter min"
+  },
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "coveralls": "^3.0.0",
+    "eslint": "^4.6.1",
+    "eslint-config-airbnb-base": "^12.0.0",
+    "eslint-config-nhsuk": "^0.8.0",
```

Comment:
All packages explicitly updated to latest

Reply:
This will need updating to the latest version of the config once the change has been merged. See https://github.com/nhsuk/eslint-config-nhsuk/pull/22 for additional information